### ISSUE TITLE
[Feature Request] Show all recipes when click on the recipe category's name

### ISSUE THREAD

**Issue by kappa-maintainer** [GTNewHorizons/NotEnoughItems#336]
Sometimes I just want to take a look at all existing crafting methods in the pack

**Comment by Abalieno** [comment]
I think you can do the same by pressing "U" on the machine/category itself.

**Comment by kappa-maintainer** [comment]
No, that's all recipes of ONE crafting method, not all recipes of ALL crafting method
You can check jei for detail
